dali/test/python/test_operator_warp.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import nose_utils
```

Comment:
# noqa - this import has a side effect and is required for Python 3.10

Reply:
Done